### ISSUE TITLE
hls_sponge action build fails for xcku15p

### ISSUE THREAD

**Issue by boekholt** [open-power/snap#755]
When building for `xcku15p-ffva1156-2-i` HLS build for hls_sponge creates the log file message
```
WARNING: [SCHED 204-21] Estimated clock period (4.1062ns) exceeds the target (target clock period: 4ns, clock uncertainty: 0.5ns, effective delay budget: 3.5ns).
WARNING: [SCHED 204-21] The critical path consists of the following:
        'call' operation ('l', action_sponge.cpp:77) to 'test_hexdigit' (3.34 ns)
        'add' operation ('tmp_s', action_sponge.cpp:80) (0.765 ns)
```
This leads to a build fail with target `check` in `actions/hls.mk`.

**Comment by bmesnet** [comment]
Corrected by Pull request #762

**Comment by boekholt** [comment]
Fixed with #762 - closing.
